saleor/static/scss/layouts/_product-page.scss

```diff
@@ -62,6 +62,9 @@
         padding: 0 2rem;
         font-style: italic;
       }
+      img {
+        width: 100%;
```

Comment:
This should be `max-width` though. User may not want to take the whole space.

Reply:
You‘re absolutely right. Thanks for reminding me that!